flink-dist/src/main/flink-bin/bin/jobmanager.sh

```diff
@@ -51,6 +51,10 @@ if [[ $STARTSTOP == "start" ]] || [[ $STARTSTOP == "start-foreground" ]]; then
         args+=("--webui-port")
         args+=("${WEBUIPORT}")
     fi
+
+    if [ ! -z "${DYNAMIC_PARAMETERS+x}" ]; then
+        for v in ${DYNAMIC_PARAMETERS[@]}; do args+=($v); done
```

Comment:
Why do we have to add it in a loop to `args`? Is this some bash magic?

Reply:
When I implemented it firstly, I ran into issues. Hence, I used the for loop option. I addressed it again and it worked. No clue, what was wrong in the first place. 🤔 Anyway, I fixed it. Now, the `args` array simply gets concatenated with the `DYNAMIC_PARAMETERS` array.